doc/api/dgram.md

```diff
@@ -177,6 +178,11 @@ system will attempt to listen on all addresses. Once binding is
 complete, a `'listening'` event is emitted and the optional `callback`
 function is called.
 
+The `options` object may contain an `fd` property. When a `fd` greater
```

Comment:
A nit: I am not sure what is correct: `` an `fd` `` or `` a `fd` ``, but these two fragments need to be unified)

Reply:
Fixed :)